Bio/File.py

```diff
@@ -171,10 +177,12 @@ def read(self, size=-1):
         return saved + self._handle.read(size)
 
     def saveline(self, line):
+        """Save the line of the File in memory."""
```

Comment:
OK. How about this then, trying to make this explicit as the ``saveline`` is the key functionality of the ``UndoHandle`` (along with ``peekline``):

```python
     def saveline(self, line):
        """Store a line in the cache memory for later use.

        This acts to undo a readline, reflecting the name of the class: UndoHandle.
        """
```